packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -107,6 +109,7 @@ export default {
     }
 
     const [sourceDataset, targetDataset] = args.argsWithoutOptions
+    const shouldSkipHistory = !!flags['skip-history']
```

Comment:
I personally prefer
```suggestion
    const shouldSkipHistory = Boolean(flags['skip-history'])
```

but just a nitpick

Reply:
Thanks! 🙂